app/policies/admin/page_policy.rb

```diff
@@ -0,0 +1,33 @@
+# Pundit policy for page admin
+class Admin::PagePolicy < Admin::DefaultPolicy
+  attr_reader :user, :record
```

Comment:
Lines 3-8 can be removed as this behaviour is already defined in the parent class `Admin::DefaultPolicy`.